docker-compose.yml

```diff
@@ -24,15 +29,20 @@ services:
       nofile:
         soft: 262144
         hard: 262144
+    healthcheck:
+      test: wget --no-verbose --tries=1 --spider http://localhost:8123/?query=SELECT%201 || exit 1
```

Comment:
Maybe we can use `/ping` route instead? `SELECT 1` might spam query_log if the users don't explicitly turn off query logs.

Reply:
Also why `plausible_db` has custom durations and `plausible_events_db` doesn't? I'd like to keep it consistent :)

Reply:
It took 34 seconds for this healthcheck to pass on my instance.

```
[+] Running 4/4
 ⠿ Container hosting-plausible_events_db-1  Healthy     34.0s
 ⠿ Container hosting-plausible_db-1         Healthy     4.0s
 ⠿ Container hosting-plausible-1            Started     31.9s
 ⠿ Container caddy-gen                      Started     32.9s
```

Reply:
Do we need `--spider` flag?

Reply:
Here's how I think it could work: https://github.com/n0vad3v/hosting/pull/1